### PR TITLE
feat: Adds v1 of public API basics

### PR DESCRIPTION
**Pull request by juliariec**
## This PR...

Adds the first version of our public API basics notebook.

### CODE REVIEW

File: public_api/public_api_basics.ipynb [hunk 1]
```diff
@@ -0,0 +1,531 @@
+{
```
Comment (svitkin) [inline public_api/public_api_basics.ipynb:1]: <p>why does it access it as <code>df['id'][3]</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SystemInternal/SystemCookbook/pull/1/discussion/#comment-1681232387'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='9'/>

Comment (svitkin) [inline public_api/public_api_basics.ipynb:1]: <p>i think there may be a benefit in creating a separate print function from the search</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SystemInternal/SystemCookbook/pull/1/discussion/#comment-1681232389'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='11'/>

Comment (svitkin) [inline public_api/public_api_basics.ipynb:1]: <p>this doesn't return the topic</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SystemInternal/SystemCookbook/pull/1/discussion/#comment-1681232390'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='13'/>

**Review by svitkin — COMMENTED** [review]
This is cool! Overall I would separate the printing and returning of objects imo. Also there's a lot of magic numbers in here for accessing ids that I think should be changed or kept consistent (they confuse me at least).

File: public_api/public_api_basics.ipynb (hunk 1 above)
Comment (vcshih) [reply public_api/public_api_basics.ipynb:1]: <p>agree with this mostly. I think we can return the df from this and then in the next block do the print functions of various parts of that data frame instead</p>

**Review by svitkin — APPROVED** [review]
Awesome! This looks great. Very [dusty]y and personal preference but you tell a cool "story" here that isn't just using the API but centered around a topic and a use-case. It could be cool to bring that more to the forefront with a couple extra comments or sentences in the markdown like "Now let's see the variables that measure osteoporosis" if that makes sense

**Pull request merged**